README.md

```diff
@@ -1,24 +1,52 @@
-Terraform Provider for Google Cloud Platform
+Terraform Provider for Google Cloud Platform (Beta)
 ==================
 
 - Website: https://www.terraform.io
+- Tutorials: [learn.hashicorp.com](https://learn.hashicorp.com/terraform?track=getting-started#getting-started)
+- Forum: [discuss.hashicorp.com](https://discuss.hashicorp.com/c/terraform-providers/tf-google/)
 - Documentation: https://www.terraform.io/docs/providers/google/index.html
 - [![Gitter chat](https://badges.gitter.im/hashicorp-terraform/Lobby.png)](https://gitter.im/hashicorp-terraform/Lobby)
 - Mailing list: [Google Groups](http://groups.google.com/group/terraform-tool)
-<img src="https://cdn.rawgit.com/hashicorp/terraform-website/master/content/source/assets/images/logo-hashicorp.svg" width="600px">
 
-Maintainers
------------
+The Terraform Google provider is a plugin for Terraform that allows for management of GCP resources.
+This provider is maintained by the [Terraform team at Google](https://cloudplatform.googleblog.com/2017/03/partnering-on-open-source-Google-and-HashiCorp-engineers-on-managing-GCP-infrastructure.html) and the Terraform team at [HashiCorp](https://www.hashicorp.com/)
 
-This provider plugin is maintained by:
+This is the ['google-beta' provider](https://github.com/hashicorp/terraform-provider-google-beta) used for preview features and features at a beta [launch stage](https://cloud.google.com/products#product-launch-stages). See [Provider Versions](https://www.terraform.io/docs/providers/google/provider_versions.html) for more details on how to use `google-beta`.
 
-* The [Google Cloud Graphite Team](https://cloudplatform.googleblog.com/2017/03/partnering-on-open-source-Google-and-HashiCorp-engineers-on-managing-GCP-infrastructure.html) at Google
-* The Terraform team at [HashiCorp](https://www.hashicorp.com/)
+Quick Starts
+----------------------
+
+- [Getting Started with the Google Provider](https://registry.terraform.io/providers/hashicorp/google/latest/docs/guides/getting_started)
+- [Provider Development](docs/contributing)
+
+Documentation
+----------------------
+
+Full, comprehensive documentation is available on the Terraform Registry:
+
+https://registry.terraform.io/providers/hashicorp/google-beta/latest/docs
+
+Upgrading the provider
+----------------------
+
+The Google provider doesn't upgrade automatically once you've started using it. After a new release you can run
+
+```bash
+terraform init -upgrade
+```
 
-Requirements
-------------
+to upgrade to the latest stable version of the Google provider. See the [Terraform website](https://www.terraform.io/docs/configuration/providers.html#provider-versions)
+for more information on provider upgrades, and how to set version constraints on your provider.
 
-- [Terraform](https://www.terraform.io/downloads.html) 0.12+
+Using the provider
```

Comment:
These 2 sections seemed to be re-written to what you have above in TPG (`Documentation` and `Upgrading the provider`), I'm guessing you meant to remove them completely here?